### DISCUSSION TITLE
Deploy self host

### DISCUSSION THREAD

**Discussion by eerison** [tinacms/tinacms#5266] [General]
Hello guys.

isn't there any docker image to use to deploy?

**Comment by kldavis4** [comment]
There's a Dockerfile in this self-hosted demo: https://github.com/tinacms/tina-self-hosted-gc-demo/blob/main/Dockerfile

**Reply by eerison** [reply]
Amazing! are there any plans to generate images? this way we do not need to fork or clone any repository, just use the image like others self host apps does!

**Reply by kldavis4** [reply]
No plans currently (cc @wicksipedia )
